assets/src/edit-story/utils/textMeasurements.js

```diff
@@ -99,3 +106,21 @@ function setStyles(node, styles) {
     }
   }
 }
+
+function changed(node, element) {
+  const lastElement = node[LAST_ELEMENT];
+  node[LAST_ELEMENT] = element;
+  if (!node.firstElementChild || !lastElement) {
+    return true;
+  }
+  return !shallowEquals(lastElement, element);
```

Comment:
This won't detect changes to e.g. padding or flip, right? Or are those property objects guaranteed immutable?

Reply:
They only change when any part of it changes. In which case I want to rerender anyway. But now that I think about it. The same applies to the element itself. I.e. it's enough for me to do `old !== new`. Will redo. Thanks for the inspiration.